client/js/condensed.js

```diff
@@ -33,5 +34,6 @@ function updateText(condensed, addedTypes) {
 			text += obj[messageType] > 1 ? "s" : "";
 		}
 	}
-	condensed.find(".condensed-msg").text(text);
+	condensed.find(".condensed-text")
+		.html(text + templates.msg_condensed_toggle());
```

Comment:
@astorije So the potential issue I see with this, is that unlike previews, condensed text can change and thus moving the button. A case can happen when you want to click the button, and someone leaves the channel at that moment and the button moves as a result of increased text.